support-frontend/assets/helpers/errorReasons.js

```diff
@@ -42,6 +42,9 @@ function appropriateErrorMessage(errorReason: ?ErrorReason): ?string {
       return 'We are unable to sign you up at this time';
     case 'internal_error':
       return 'Sorry, something has gone wrong. Please try again, or come back later.';
+    case 'card_authentication_error':
+      // TODO - is this copy ok?
+      return 'Your card issuer\'s authentication was unsuccessful and you have not been charged. Please try again or choose another payment method.';
```

Comment:
I'm thinking we don't want to encourage them to try again w the same payment method if they've gotten this specific error